sdk/JavaScript/TestFramework/js/tests/allTests.js

```diff
@@ -39,31 +39,31 @@
     zumo.testGroups.push(new zumo.Group(zumo.tests.blog.name, zumo.tests.blog.tests));
     
     ////Add addistional Win JS scenario if user run WinJS application
-    //if (!testPlatform.IsHTMLApplication) {
-    //    zumo.testGroups.push(new zumo.Group(zumo.tests.query.name + ' (server side)', zumo.tests.query.serverSideTests));
+    //if (!testPlatform.IsHTMLApplication) {
+    //    zumo.testGroups.push(new zumo.Group(zumo.tests.query.name + ' (server side)', zumo.tests.query.serverSideTests));
     //}
     zumo.testGroups.push(new zumo.Group(zumo.tests.updateDelete.name, zumo.tests.updateDelete.tests));
     zumo.testGroups.push(new zumo.Group(zumo.tests.login.name, zumo.tests.login.tests));
     zumo.testGroups.push(new zumo.Group(zumo.tests.misc.name, zumo.tests.misc.tests));
-    //if (!testPlatform.IsHTMLApplication) {
-    //    zumo.testGroups.push(new zumo.Group(zumo.tests.push.name, zumo.tests.push.tests));
-    //}
+    if (!testPlatform.IsHTMLApplication) {
```

Comment:
I could set this up for html apps as well.  However, I get the impression that moving on to v2 automation is higher priority.